package.json

```diff
@@ -32,72 +40,46 @@
     "react": ">=16.3.0"
   },
   "dependencies": {
+    "@types/sinon": "^9.0.0",
```

Comment:
could you please clarify why type definitions are not in devDependencies?

Reply:
@doctyper I think he's right, in my next push to this branch I can move to devDependencies if there are no objections